tests/test_layout.py

```diff
@@ -47,3 +47,19 @@ def test_focus_wrap(hlwm, running_clients, running_clients_num):
         assert int(hlwm.get_attr('tags.0.curframe_windex')) == expected_idx
         if i < running_clients_num - 1:
             hlwm.call('focus down')
+
+
+@pytest.mark.parametrize("path", '@ 0 1 00 11 . / /. ./'.split(' '))
+@pytest.mark.parametrize("running_clients_num", [3])
+@pytest.mark.parametrize("num_splits", [0, 1, 2])
```

Comment:
Uh nice, I wasn't aware that it would work like this. I've stolen this for the keymask tests (see #320)